kobo/apps/audit_log/views.py

```diff
@@ -321,3 +324,174 @@ class AccessLogViewSet(AuditLogViewSet):
+
+    <pre class="prettyprint">
+    <b>GET</b> /api/v2/access-logs/me/export
+    </pre>
+
+    > Example
+    >
+    >       curl -X GET https://[kpi-url]/access-logs/me/export
+
+    > Response 200
+    >
+    >       [
+    >           {
+    >               "uid": "aleooVUrhe3cRrLY5urRhxLA",
+    >               "status": "complete",
+    >               "date_created": "2024-11-26T21:27:08.403181Z"
+    >           },
+    >           {
+    >               "uid": "aleMzK7RnuaPokb86TZF2N4d",
+    >               "status": "complete",
+    >               "date_created": "2024-11-26T20:18:55.982974Z"
+    >           }
+    >       ]
+
+    ### Creates an export task
+
+    <pre class="prettyprint">
+    <b>POST</b> /api/v2/access-log/me/export
+    </pre>
+
+    > Example
+    >
+    >       curl -X POST https://[kpi-url]/access-logs/me/export
+
+    > Response 202
+    >
+    >       [
+    >           "status: created"
+    >       ]
+    >
+    """
+
+    def create(self, request, *args, **kwargs):
+        if AccessLogExportTask.objects.filter(
+            user=request.user,
+            status=AccessLogExportTask.PROCESSING,
+            get_all_logs=False,
+        ).exists():
+            return Response(
+                {
+                    'error': (
+                        'You already have a running export task for your own logs.'
+                    )
+                },
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+        return self.create_task(request, get_all_logs=False)
+
+    def list(self, request, *args, **kwargs):
+        return self.list_tasks(request.user)
+
+
+class AllAccessLogsExportViewSet(BaseAccessLogsExportViewSet):
+    """
+    Access logs export
+
+    Lists all access logs export tasks for all users. Only available to superusers.
+
+    <pre class="prettyprint">
+    <b>GET</b> /api/v2/access-logs/export
+    </pre>
+
+    > Example
+    >
+    >       curl -X GET https://[kpi-url]/access-logs/export
+
+    > Response 200
+    >
+    >       [
+    >           {
+    >               "uid": "aleooVUrhe3cRrLY5urRhxLA",
+    >               "status": "complete",
+    >               "date_created": "2024-11-26T21:27:08.403181Z"
+    >           },
+    >           {
+    >               "uid": "aleMzK7RnuaPokb86TZF2N4d",
+    >               "status": "complete",
+    >               "date_created": "2024-11-26T20:18:55.982974Z"
+    >           }
+    >       ]
+
+    ### Creates an export task
+
+    <pre class="prettyprint">
+    <b>POST</b> /api/v2/access-log/export
+    </pre>
+
+    > Example
+    >
+    >       curl -X POST https://[kpi-url]/access-logs/export
+
+    > Response 202
+    >
+    >       [
+    >           "status: created"
+    >       ]
+    >
+    """
+
+    permission_classes = (SuperUserPermission,)
+
+    def create(self, request, *args, **kwargs):
+        # Check if the superuser has a task running for all or just their own logs
```

Comment:
```suggestion
        # Check if the superuser has a task running for all
```
since we don't check to see if they're looking for their own logs